03-transactions.md

```diff
@@ -12,7 +12,7 @@ This details the exact format of on-chain transactions, which both sides need to
         * [Commitment Transaction Outputs](#commitment-transaction-outputs)
           * [`to_local` Output](#to_local-output)
           * [`to_remote` Output](#to_remote-output)
-          * [`to_local_anchor` and `to_remote_anchor`](#to_local_anchor-and-to_remote_anchor-output-option_anchor_outputs)
+          * [`to_local_anchor` and `to_remote_anchor`](#to_local_anchor-and-to_remote_anchor-output-option_anchors)
```

Comment:
have you tried to remove this and check if the CI will fail?

Reply:
it also did not catch this one

Reply:
Amazing a url checker that is not able to check the relative url! so maybe you can run your scrip inside the CI?